### DISCUSSION TITLE
Three states for collapsible side-bar

### DISCUSSION THREAD

**Discussion by almereyda** [outline/outline#7098] [Ideas]
When working in multiple Outline instances with multiple organisations, it might be helpful to retain a certain visual cue to quickly differentiate the contexts one in working in.

For this we have a large side bar that is both visually appealing and useful. The appeal comes in parts from its careful use of iconography and colour.

When reducing the visual demand from the application by collapsing the sidebar, it becomes invisible and returns optionally when hovering the left side.

Yet in some situations it might be supportive to have a visually and locally unique appearance of an instance, to provide the user with contextual cues for their current environment.

Long story short, how would everybody like a collapsible sidebar of sorts, which keeps the icons visible?

![Screenshot 2024-06-20 at 22-59-37 Database embedding - Allmende Docs](https://github.com/outline/outline/assets/1645308/7152cdb6-0844-4218-964d-11667d916a29)

In hindsight, it's actually what Notea does. Which doesn't know to hide the sidebar in return. See https://github.com/notea-org/notea/issues/185 for visual comparison.

Maybe three states in the collapsible side-bar can be useful?

- expanded
- compressed
- hidden

**Comment by tommoor** [comment]
I've considered this before and decided against it at that time. I think this kind of sidebar can work with just a few icons – but with many like in Outline you end up needing to have some sort of hover display for each icon as you pass over for it to be usable. At that point it just becomes kind of a bigger version of the existing collapsed sidebar.

**Reply by almereyda** [reply]
Yes, indeed. Now that you say it.

All hovering to reveal useful interaction points and information also poses challenges for touch users.
